apps/ledger-live-desktop/src/renderer/screens/manager/DeviceDashboard/DeviceInformationSummary/DeviceName.tsx

```diff
@@ -48,6 +50,9 @@ const DeviceName: React.FC<Props> = ({
   );
 
   const openDeviceRename = useCallback(() => {
+    // Prevents manager from reacting to device changes
+    setPreventResetOnDeviceChange(true);
```

Comment:
`setPreventResetOnDeviceChange(false);` is never called ?
I would suggest to add this call to the `onClose` prop: `setDrawer(EditDeviceName, {onClose: () => {/*... here */}})`

Reply:
Done 👍